README.md

```diff
@@ -38,7 +38,9 @@ render(<App />, document.getElementById('root'));
 ### [ModalComponent, openFunc, closeFunc, isOpenBool] = useModal(domNode?, { preventScroll?, focusTrapOptions?, showCloseButton?, renderCloseButton? })
 
 `ModalComponent`
+Type: React.FC<{ title?: React.ReactNode; description?: React.ReactNode, children?: React.ReactNode }>
 Modal component that displays children in the screen center.
+If you specify `title` and `description`, you must implement custom components with the `components` option's `Modal` property and render in them.
```

Comment:
If the components option is not used, the default components will be used.
This is because it would be difficult to have a component that renders the default title,description.